frontend/public/components/secrets/create-secret/PullSecretCredentialEntry.tsx

```diff
@@ -34,92 +45,72 @@ export const PullSecretCredentialEntry: React.FC<PullSecretCredentialEntryProps>
-          </span>
-        </div>
-      </div>
-      <div className="form-group">
-        <label htmlFor={`${id}-email`}>{t('public~Email')}</label>
-        <div>
-          <span className="pf-v6-c-form-control">
-            <input
-              id={`${id}-email`}
-              type="text"
-              name="email"
-              onChange={handleChangeEvent}
-              value={email}
-              onBlur={handleBlurEvent}
-              data-test="image-secret-email"
-            />
-          </span>
-        </div>
-      </div>
-    </PaneBodyGroup>
+    <FormFieldGroup data-test-id="create-image-secret-form">
+      <FormGroup label={t('public~Registry server address')} isRequired fieldId={`${id}-address`}>
+        <TextInput
+          id={`${id}-address`}
+          type="text"
+          name="address"
+          value={address}
+          onChange={(_event, value) => updateEntry('address', value)}
+          onBlur={(event) => updateEntry('address', event.currentTarget.value.trim())}
+          data-test="image-secret-address"
+          isRequired
+        />
+        <FormHelperText>
+          <HelperText>
+            <HelperTextItem>{t('public~For example quay.io or docker.io')}</HelperTextItem>
+          </HelperText>
+        </FormHelperText>
+      </FormGroup>
+      <FormGroup label={t('public~Username')} isRequired fieldId={`${id}-username`}>
+        <TextInput
+          id={`${id}-username`}
+          type="text"
+          name="username"
+          value={username}
+          onChange={(_event, value) => updateEntry('username', value)}
+          onBlur={(event) => updateEntry('username', event.currentTarget.value.trim())}
+          data-test="image-secret-username"
+          isRequired
+        />
+      </FormGroup>
+      <FormGroup label={t('public~Password')} isRequired fieldId={`${id}-password`}>
+        <TextInput
+          id={`${id}-password`}
+          type="password"
+          name="password"
+          value={password}
+          onChange={(event, value) => updateEntry('password', value)}
+          onBlur={(event) => updateEntry('password', event.currentTarget.value.trim())}
+          data-test="image-secret-password"
+          isRequired
+        />
+      </FormGroup>
+      <FormGroup label={t('public~Email')} fieldId={`${id}-email`}>
+        <TextInput
+          id={`${id}-email`}
+          type="email"
+          name="email"
+          value={email}
+          onChange={(_event, value) => updateEntry('email', value)}
+          onBlur={(event) => updateEntry('email', event.currentTarget.value.trim())}
+          data-test="image-secret-email"
+        />
+      </FormGroup>
+      {showRemoveButton && (
+        <Button
+          onClick={() => removeEntry(id)}
+          type="button"
+          variant="link"
+          data-test="remove-entry-button"
+          icon={<MinusCircleIcon />}
+        >
+          {t('public~Remove credentials')}
+        </Button>
+      )}
```

Comment:
same comment about wrapping button in ActionsGroup to make the button not so long